src/helpers/logger.js

```diff
@@ -0,0 +1,25 @@
+import Constants from '../config/constants';
+
+/**
+ * Logger class
+ */
+class Logger {
+    /**
+     * Reusable log function, can be used to display a detailed logging info.
+     * @param {*} message
+     * @param {*} level
+     */
+    static log(message, level = 'ERROR') {
```

Comment:
Wouldn't it be better to have 3 methods file info, warning, error rather than having a function with level field?, and why did you use `let` over `const`?

Reply:
Then it would repeat the same method with just a different parameter. Rather wouldn’t it be better if we could come up with a better name for the level parameter and when you calling the method, if it’s a INFO or a WARNING, just specify it in the parameter list and let ERROR to be the default one. 
And yes we can use const there.. I’ll change that.